src/reanimated2/hook/useComposedEventHandler.ts

```diff
@@ -0,0 +1,52 @@
+import { useEvent } from './useEvent';
+import { useHandler } from './useHandler';
+import { WorkletEventHandler } from '../WorkletEventHandler';
+import type { ReanimatedEvent } from './commonTypes';
+import type { WorkletFunction } from '../commonTypes';
+
+/**
+ * Lets you compose multiple event handlers based on [useEvent](https://docs.swmansion.com/react-native-reanimated/docs/advanced/useEvent) hook.
+ *
+ * @param handlers - A list of event handlers created using [useEvent](https://docs.swmansion.com/react-native-reanimated/docs/advanced/useEvent) hook.
+ * @returns An object you need to pass to any "onEvent" prop on an `Animated` component.
+ */
+export function useComposedEventHandler<
+  Context extends Record<string, unknown>
+>(handlers: { workletEventHandler: typeof WorkletEventHandler }[]) {
+  const workletsRecord: Record<string, WorkletFunction> = {};
```

Comment:
Would be nice to annotate here that we use an object here for uniformity with `useHandler`.